wpeverest/everest-forms
# Enhancement - Drop-down date option added

### All Submissions:






### Changes proposed in this Pull Request:



This PR allows user to apply drop-down date from the field setting. 

### How to test the changes in this Pull Request:

1. Add date field to the panel.
2. Choose date as date style form field.
3. Apply various options under advance setting.
4. Verify the changes..

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Drop-down date option added
